oar/core/notification.py

```diff
@@ -514,10 +514,10 @@ def get_slack_message_for_unhealthy_advisories(self, unhealthy_advisories):
             str: slack message
         """
         gid = self.sc.get_group_id_by_name(
-            self.cs.get_slack_user_group_from_contact_by_id("art")
+            self.cs.get_slack_user_group_from_contact_by_id("ert")
```

Comment:
@ming1013 there is no contact called `ert` in config store, the requirement of this change is: send mesage to release lead instead. please refer to https://github.com/openshift/release-tests/blob/master/oar/core/notification.py#L542-L546